ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -78,6 +78,19 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-ipi-tolatest-f28
+  cron: 25 4 4 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - ref: cucushift-upgrade-setchannel
+    - ref: cucushift-ota-preupgrade
+    - ref: cucushift-upgrade-toversion
```

Comment:
could u help add a comment `# ENABLE_OTA_TEST: OCP-23799` in this step so that we could grep it easily? for example, this line should be a proper place. https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/toversion/cucushift-upgrade-toversion-commands.sh#L347

Reply:
added comment at https://github.com/openshift/release/pull/47434/files#diff-157cbcc242fbd960323cfa685f2c66b667a8b42f39bf4d860bce52a22c7d2a99R314